scripts/check-lockfile-version.mjs

```diff
@@ -0,0 +1,22 @@
+/**
+ * Ensure all package-lock.json are using a lockVersion equal or greater than 3.
+ */
+import {readFileSync} from 'node:fs';
+import {resolve} from 'node:path';
+
+const lockFileVersionFinder = /(?<="lockfileVersion":\s)\d(?=,)/;
+
+const lockFilePaths = process.argv[2].toString().split('\n');
+let err = '';
+for (const lockFilePath of lockFilePaths) {
+  console.log(lockFilePath);
+  const lockFileText = readFileSync(resolve(lockFilePath)).toString();
+  // Package-lock.json are bulky and the info we need is right at the top of the file, so regex-match to go fast.
```

Comment:
can readSync be used with `length` arg instead for better perf?

Reply:
If I want to keep the script generic, not really.
The first thing I thought of was going async and use a pipe to detect when we have the data, but same thing: cannot be sure that the data I got at a given time contains everything, so I can have let's say `endofthepackagename",\n\t"lockfi` followed by `"leVersion": 3,`

At this stage, I think the complexity is not worth the gain in speed.

Reply:
Makes sense 👍